tests/integration/test_charm.py

```diff
@@ -187,3 +188,45 @@ async def test_with_ingress(
     )
     assert response.status_code == 200
     assert "Hello, World!" in response.text
+
+
+@pytest.mark.parametrize(
+    "endpoint,db_name, db_channel",
+    [
+        (
+            "mysql/status",
+            "mysql-k8s",
+            "8.0/stable",
```

Comment:
If the trailing comma on this line is deleted `black` will probably reduce the number of lines this is spread over

Reply:
Fixed